src/views/InvolvementsAll/index.jsx

```diff
@@ -80,6 +84,10 @@ const InvolvementsAll = () => {
     navigate(`?session=${value}`);
   };
 
+  const handleChange = (event, newValue) => {
```

Comment:
I would rename this to be a little bit more descriptive of what change it is handling. Perhaps `handleChangeTab(...)`?